apps/fyler/src/fyler_sup.erl

```diff
@@ -77,6 +91,13 @@ init([]) ->
         supervisor,
         []
       },
+      {callback_sup,
+        {supervisor,start_link,[{local, callback_sup}, ?MODULE, [hackney_worker]]},
```

Comment:
Line 95 is too long:         {supervisor,start_link,[{local, callback_sup}, ?MODULE, [hackney_worker]]},.